workspaces/admin.py

```diff
@@ -22,13 +22,20 @@ class WorkspaceInviteAdmin(admin.ModelAdmin):
         "email",
         "status",
         "has_expired",
+        "clicks",
         "created_by",
         "updated_by",
         "created_at",
         "updated_at",
     ]
     list_filter = ["created_at", "status"]
-    readonly_fields = ["auto_accepted", "has_expired", "created_at", "updated_at"]
+    readonly_fields = [
+        "clicks",
+        "auto_accepted",
+        "has_expired",
+        "created_at",
+        "updated_at",
+    ]
```

Comment:
@nikochiko thoughts?

Reply:
i thought we might not need it in the inline view at all

Reply:
This would make the invites hard to monitor. We need the view to track invites on django and I think that view is the `WorkspaceInviteInline ` right now